### PR TITLE
Changes to Splunk

### PR DESCRIPTION
**Pull request by ts3ng**
This will replace #79 with additional fixes. merged with latest master changes as well. Deployment was tested in booster sandbox environment.

Changelog

Additional fields added to system logging
fixes Re-establish journalpipe connection for system logging
fixes to journalpipe system log service.
journalpipe format changed from json to raw
Reorganized all splunk units to be optional
fixes to logging-config fleet unit
reorganized setup scripts. ## Notes
You must explain how any docker run executions your PR introduces handles log accumulation.

Did you introduce any command that executes docker run?
No
Are there any dependencies on github.com/adobe-platform/infrastructure?
Secrets.
If so:

Is an update to the base stack required(rare)?
No
Is an A/B required?
Yes

Are you dependent on new secrets, or infrastructure in any way?
Yes, infrastructure will.

### CODE REVIEW

**Comment by ts3ng** [comment]
@matthewdfuller this is ready to be merged.  This includes some critical fixes that are wanted by ops for system logs, as well as changes to the customer logging logging tier units.

File: v1/opt/splunk/fleet/journald-splunk-pipe.service [hunk 1]
```diff
@@ -4,16 +4,13 @@ After=bootstrap.service splunk-journald.service
 Wants=splunk-journald.service
 
 [Service]
-EnvironmentFile=/etc/environment
-Environment="INSTANCEID=curl -s http://169.254.169.254/latest/meta-data/ami-id"
-Environment="HOSTNAME=curl -s http://169.254.169.254/latest/meta-data/hostname"
-Environment="STACK_NAME=etcdctl get /environment/STACK_NAME"
+EnvironmentFile=/etc/splunk.env
```
Comment (matthewdfuller) [inline v1/opt/splunk/fleet/journald-splunk-pipe.service]: When does this get written? Maybe it was in an earlier PR and I just missed it

Comment (matthewdfuller) [inline v1/opt/splunk/fleet/journald-splunk-pipe.service]: Perfect, thanks!

**Comment by ts3ng** [comment]
@matthewdfuller thanks matt, feel free to merge when you have time.  @manojbadam will need some of these changes are it includes some fixes for the system logging.

**Pull request merged**